ios/ReactTestApp/React+Compatibility.m

```diff
@@ -25,3 +48,37 @@ void RTATriggerReloadCommand(RCTBridge *bridge, NSString *reason)
     RCTTriggerReloadCommandListeners(reason);
 #endif
 }
+
+// MARK: - [0.63.2] Images do not render on iOS 14
+// See https://github.com/facebook/react-native/pull/29420
+
+#if !TARGET_OS_OSX && REACT_NATIVE_VERSION < 6302
+
+#import <React/RCTUIImageViewAnimated.h>
+
+@implementation RCTUIImageViewAnimated (ReactTestApp)
+
++ (void)initialize
+{
+    if ([self class] != [RCTUIImageViewAnimated class]) {
```

Comment:
I think initialize can be called more than once if there are other subclasses or categories on RCTUIImageViewAnimated.   Put inside dispatch_once ?

Reply:
Apparently, methods get (unreliably) overridden when redefined in categories: https://stackoverflow.com/questions/5272451/overriding-methods-using-categories-in-objective-c. In which case, the if-check should be enough. I've added `dispatch_once` anyway just to be safe.